www/app/App.data.js

```diff
@@ -0,0 +1,11 @@
+import seoImage from '../shared/media/images/seo.png';
+
+const SEO_DATA = {
```

Comment:
Use `export default { ...` since each _data_ file should only export one _data_ object.